rosdep/base.yaml

```
@@ -1451,6 +1451,14 @@ glpk:
   gentoo: [sci-mathematics/glpk]
   rhel: [glpk-devel]
   ubuntu: [libglpk-dev]
+glslc:
+  arch: [shaderc]
+  debian: [glslc]
+  fedora: [glslc]
+  gentoo: [media-libs/shaderc]
+  ubuntu:
+    '*': null
+    noble: [glslc]
```

We tend to do this the other way, with the thinking that if it is in `noble`, it will also be in later versions.  This means we won't have to update the key for the next release:

```suggestion
    '*': [glslc]
    focal: null
    jammy: null
```